src/main/java/com/coveo/spillway/storage/RedisStorage.java

```diff
@@ -65,96 +62,9 @@
   private final JedisPool jedisPool;
   private final String keyPrefix;
 
-  public RedisStorage(URI uri) {
```

Comment:
This is technically a breaking change... shouldn't you tag those as @deprecated instead until we release a major version? (I know I'm a PITA)